package.sh

```diff
@@ -6,6 +6,6 @@ for GOOS in darwin linux windows; do
         echo "Building ${architecture}"
         export GOOS=$GOOS
         export GOARCH=$GOARCH
-        go build -ldflags "-X ${PROJECT_PATH}/${PROJECT_NAME}/update.architecture=${architecture} -X ${PROJECT_PATH}/${PROJECT_NAME}/update.urlUpdateRelease=${URL_UPDATE_RELEASE} -X ${PROJECT_PATH}/${PROJECT_NAME}/update.urlUpdateSnapshot=${URL_UPDATE_SNAPSHOT}" -o bin/sail-${architecture}
+        go build -ldflags "-X ${PROJECT_PATH}/${PROJECT_NAME}/update.architecture=${architecture} -X ${PROJECT_PATH}/${PROJECT_NAME}/update.urlUpdateSnapshot=${URL_UPDATE_SNAPSHOT}" -o bin/sail-${architecture}
```

Comment:
urlUpdateRelease is always github, so no need to set it in package.sh
urlUpdateSnapshot can be set on package.sh